tests/infra/ccf.py

```diff
@@ -269,7 +270,7 @@ def _add_node(self, node, lib_name, args, target_node=None, should_wait=True):
         )
 
         # If the network is opening, node are trusted without consortium approval
-        if self.status == ServiceStatus.OPENING and should_wait:
+        if self.status == ServiceStatus.OPENING and should_wait and not args.pbft:
```

Comment:
`should_wait` was already added to open the network with PBFT so I don't think both are necessary.